modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/src/Plugin/Block/BranchContactsInfo.php

```diff
@@ -0,0 +1,138 @@
+
+  /**
+   * @var \Drupal\pathauto\AliasCleaner
+   */
+  protected $alias_cleaner;
+
+  /**
+   *   Plugin config
+   *
+   * @param array $configuration
+   *   Plugin id
+   * @param string $plugin_id
+   *   Plugin definition
+   * @param mixed $plugin_definition
+   *   RouteMatch service instance
+   * @param $routeMatch
+   *   Renderer service instance
+   * @param $renderer
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, $routeMatch, $renderer, $alias_cleaner) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->routeMatch = $routeMatch;
+    $this->renderer = $renderer;
+    $this->alias_cleaner = $alias_cleaner;
+  }
+
+  /**
+   * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+   * @param array $configuration
+   * @param string $plugin_id
+   * @param mixed $plugin_definition
+   *
+   * @return static
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('current_route_match'),
+      $container->get('renderer'),
+      $container->get('pathauto.alias_cleaner')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $render_array = [];
+
+    $node = $this->routeMatch->getParameter('node');
+    if ($node instanceof NodeInterface && $node->getType() == 'branch') {
+      $render_array = ['#theme' => 'block_branch_contacts_info'];
+
+      $address_array = $node->get('field_location_address')->get(0)->toArray();
+      $location_address = "{$address_array['address_line1']} {$address_array['locality']}, {$address_array['administrative_area']} {$address_array['postal_code']}";
+      $directions_url = Url::fromUri('https://www.google.com/maps/dir/', [
+        'query' => [
+          'api' => 1,
+          'destination' => $this->alias_cleaner
+            ->cleanString($location_address),
+        ],
+      ])->toString();
+
+      $render_array['#branch_title'] = $node->getTitle();
+      $render_array['#phone'] = $node->get('field_location_phone')
+        ->get(0)
+        ->getString();
+      $render_array['#address_title'] = $location_address;
+      $render_array['#directions_url'] = $directions_url;
+
+      $branch_selector = openy_branch_selector_get_link($node->id());
+      $render_array['#openy_branch_selector'] = $this->renderer->render($branch_selector);
+
+      $branch_hours = $node->get('field_branch_hours')->view([
+        'type' => 'openy_today_custom_hours',
+        'settings' => [],
+      ]);
+      $render_array['#branch_hours'] = $this->renderer->render($branch_hours);
+    }
+
+    return $render_array;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
+    if ($node = $this->routeMatch->getParameter('node')) {
+      //if there is node add its cachetag
```

Comment:
ensure you have Drupal style comments settings in IDE.
Should be 
`// If there is a node add its cachetag.`
Not blocking release